README.md

```diff
@@ -0,0 +1,12 @@
+## Drawing Sheet WebPage
+#### This WebApp contains-
+>Seven colors from Red, Yellow, Green, Blue, Orange, Pink and Indigo can be used to color the webpage sheet.
+>Three different size strokes of these colors
+
+#### How to Run-

```

Comment:
### How to Run

Add this point in the current README  
All other points are already present😄